src/components/QueryEditor.tsx

```diff
@@ -156,6 +156,7 @@ export function QueryEditor(props: Props) {
             aria-labelledby={`${props.query.refId}-wait`}
             onChange={onWaitForChange}
             checked={query.waitForResult}
+            css=""
```

Comment:
Why does this do?

Reply:
No clue. The `css` property is now required so I had to add something 🤷‍♀️  I figured an empty string was safe.

I'll look a bit further into it, in the meantime if someone has any idea (maybe @asimpson as you've been involved in switch component recently?) feel free to chime in!

We've got time, the release will be done after Grafana 9 stable 😃

Reply:
Had a chat with @sunker and after a quick search quite a lot of Grafana plugins pass `css=""` along. 

So we're safe to merge this PR ... although this although looks like a bug / useless prop to have required.

I'll have a look at further version of grafana-ui see if the most recent ones don't require the prop

Reply:
Had a chat with @sunker and after a quick search quite a lot of Grafana plugins pass `css=""` along. 

So we're safe to merge this PR ... although this although looks like a bug / useless prop to have required.

I'll have a look at further version of grafana-ui see if the most recent ones don't require the prop

Reply:
Upgrading worked (thanks @sunker ) so useless prop not needed anymore 😉